apps/backend/src/user/user.controller.ts

```diff
@@ -36,6 +36,13 @@ export class UserController {
   async updateCurrentUser(@Body() updateUserDto: UpdateUserDto, @CurrentUser() user: User) {
     return this.userService.update(user.authSchId, updateUserDto);
   }
+  @Get('members')
+  async getMembers(
+    @Query('page', ParseIntPipe) page: number = 1,
```

Comment:
if the pageNumber is 1, we're skipping the first page and get the second page. Either subtract 1 in the service or make the default value 0